lib/ansible/modules/cloud/podman/podman_container.py

```diff
@@ -0,0 +1,1350 @@
+      - I(present) - Asserts the existence of a container matching the name and any provided configuration parameters. If no
+          container matches the name, a container will be created. If a container matches the name but the provided configuration
+          does not match, the container will be updated, if it can be. If it cannot be updated, it will be removed and re-created
+          with the requested config. Image version will be taken into account when comparing configuration. Use the recreate option
+          to force the re-creation of the matching container.
+      - I(started) - Asserts there is a running container matching the name and any provided configuration. If no container
+          matches the name, a container will be created and started. Use recreate to always re-create a matching container, even
+          if it is running. Use force_restart to force a matching container to be stopped and restarted.
+      - I(stopped) - Asserts that the container is first I(present), and then if the container is running moves it to a stopped
+          state.
+    type: str
+    default: 'started'
+    choices:
+      - absent
+      - present
+      - stopped
+      - started
+  image:
+    description:
+      - Repository path (or image name) and tag used to create the container. If an image is not found, the image
+        will be pulled from the registry. If no tag is included, C(latest) will be used.
+      - Can also be an image ID. If this is the case, the image is assumed to be available locally.
+    type: str
+  annotation:
+    description:
+      - Add an annotation to the container. The format is key value, multiple times.
+    type: dict
+  authfile:
+    description:
+      - Path of the authentication file. Default is ``${XDG_RUNTIME_DIR}/containers/auth.json``
+        (Not available for remote commands) You can also override the default path of the authentication file
+        by setting the ``REGISTRY_AUTH_FILE`` environment variable. ``export REGISTRY_AUTH_FILE=path``
+    type: str
+  blkio_weight:
+    description:
+      - Block IO weight (relative weight) accepts a weight value between 10 and 1000
+    type: int
+  blkio_weight_device:
+    description:
+      - Block IO weight (relative device weight, format DEVICE_NAME[:]WEIGHT).
+    type: str
+  cap_add:
+    description:
+      - List of capabilities to add to the container.
+    type: list
+  cap_drop:
+    description:
+      - List of capabilities to drop from the container.
+    type: list
+  cgroup_parent:
+    description:
+      - Path to cgroups under which the cgroup for the container will be created.
+        If the path is not absolute, the path is considered to be relative to
+        the cgroups path of the init process. Cgroups will be created if they do not already exist.
+    type: str
+  cidfile:
+    description:
+      - Write the container ID to the file
```

Comment:
If the container is already running and this option is specified, will the module make sure that the correct ID is contained in the file? If not, this should be properly documented.